x-pack/legacy/plugins/siem/public/lib/compose/kibana_compose.tsx

```diff
@@ -7,29 +7,26 @@
 import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory';
 import ApolloClient from 'apollo-client';
 import { ApolloLink } from 'apollo-link';
-import 'ui/autoload/all';
```

Comment:
After a bit of git spelunking I'm fairly sure these lines are unnecessary, but I'd appreciate a sanity check on that.

Reply:
Yes that was there to bootstrap with map embeddable at the beginning